src/test/java/org/opensearch/ad/model/ModelProfileTests.java

```diff
@@ -0,0 +1,43 @@
+package org.opensearch.ad.model;
+
+import static org.opensearch.common.xcontent.XContentFactory.jsonBuilder;
+
+import java.io.IOException;
+
+import org.opensearch.ad.AbstractADTest;
+import org.opensearch.ad.constant.CommonName;
+import org.opensearch.common.Strings;
+import org.opensearch.common.xcontent.ToXContent;
+import org.opensearch.common.xcontent.XContentBuilder;
+
+import test.org.opensearch.ad.util.JsonDeserializer;
+
+public class ModelProfileTests extends AbstractADTest {
+
+    public void testToXContent() throws IOException {
+        ModelProfile profile1 = new ModelProfile(
+                randomAlphaOfLength(5),
+                Entity.createSingleAttributeEntity(randomAlphaOfLength(5), randomAlphaOfLength(5)),
+                0
```

Comment:
Why are we keeping modelSize as 0 here? Let's assign some model size

Reply:
The constructer has a check where it validates the modelSizeInBytes variable and if it is greater than 0 then only it will add the field in build. The test is written to validate that field.